pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+            var addLibDlg = ShowDialog<EditLibraryDlg>(libListDlg.AddItem);
+            var drosophilaLipids = "Drosophila Lipids";
+            RunUI(() =>
+            {
+                //   �	Enter �Drosophila Lipids� in the Name field of the Edit Library form.
+                addLibDlg.LibraryName = drosophilaLipids;
+                //   �	Click the Browse button.
+                //   �	Navigate to the SmallMoleculeLibraries folder created earlier.
+                //   �	Select the �Drosophila_Lipids_Neg.blib� file.
+                addLibDlg.LibraryPath = TestFilesDirs[0].GetTestPath("Drosophila_Lipids_Neg.blib");
+            });
+            //   �	Click the Open button.
+            //   �	Click the OK button in the Edit Library form.
+            OkDialog(addLibDlg, addLibDlg.OkDialog);
+            //   �	Click the OK button in the Edit Libraries form.
+            OkDialog(libListDlg, libListDlg.OkDialog);
+
+            //   The Libraries list in the Molecule Settings form should now contain the Drosophila Lipids library you just created.
+            //   �	Check the Drosophila Lipids checkbox to tell Skyline to use this library in the current document.
+            //   �	If you have any other libraries in this list checked, uncheck them now.
+            RunUI(() => { peptideSettingsUI.PickedLibraries = new[] {drosophilaLipids}; });
+            //   The Molecule Settings form should now look like:
+            PauseForScreenShot("Molecule Settings - Library", 7);
+            //   �	Click the OK button in the Molecule Settings form.
+            OkDialog(peptideSettingsUI, peptideSettingsUI.OkDialog);
+            doc = WaitForDocumentChangeLoaded(doc);
+
+            //   To open the library explorer and view the contents of the library you just added, do the following: 
+            //   �	From the View menu, click Spectral Libraries.
+            var viewLibUI = ShowDialog<ViewLibraryDlg>(SkylineWindow.ViewSpectralLibraries);
+
+            //   The library explorer should now resemble the image below:
+            PauseForScreenShot("Library Explorer (probably need to resize wider)", 8);
+
+            //  To add all the molecules in the library to your target list:
+            //   �	Click the Add All button.
+            //   �	A popup window will then notify you that you will add 34 molecules, 38 precursors, and 246 transitions to the document. Click Add All.
+            var confirmAddDlg = ShowDialog<AlertDlg>(viewLibUI.AddAllPeptides);
+            using (new CheckDocumentState(1, 34, 38, 246))
+            {
+                RunUI(confirmAddDlg.OkDialog);
+                doc = WaitForDocumentChangeLoaded(doc);
+            }
+
+            //   �	Close the Spectral Library Explorer window.
+            OkDialog(viewLibUI, viewLibUI.CancelDialog);
+
+            //   Your Skyline window should now resemble:
+            PauseForScreenShot("Populated Skyline window", 9);
```

Comment:
Usually want to resize SkylineWindow to an exact size before the first full window screenshot.